sound/soc/sof/ops.h

```diff
@@ -21,6 +21,15 @@
 #define sof_ops(sdev) \
 	((sdev)->pdata->desc->ops)
 
+#define sof_dsp_ipc_ops(sdev) \
+	((sdev)->pdata->desc->ops->ipc_ops)
```

Comment:
can we drop the _ops suffix from ipc_ops, fw_ops and trace_ops? ((sdev)->pdata->desc->ops->ipc) already has ops in it